examples/dreambooth/README.md

```diff
@@ -22,7 +22,7 @@ accelerate config
 
 ### Dog toy example
 
-You need to accept the model license before downloading or using the weights. In this example we'll use model version `v1-4`, so you'll need to visit [its card](https://huggingface.co/CompVis/stable-diffusion-v1-4), read the license and tick the checkbox if you agree. 
+You need to accept the model license before downloading or using the weights. In this example we'll use model version `v1-4`, so you'll need to visit [its card](https://huggingface.co/runwayml/stable-diffusion-v1-5), read the license and tick the checkbox if you agree. 
```

Comment:
I'm not sure if we want to change the model in examples. Think this is up to the user. The commands have specific hprams tested with specific checkpoint, and are supposed to to work with that checkpoint out of the box, in case some users want to just follow the example.

Reply:
IMO we would like to have the latest model in the examples. What kind of hparams are tuned to v1-4? I remember when we released diffusers we were working on v1-3 and v1-4 dropped quite late, did we swap a lot of stuff ?

Reply:
+1 on Suraj here

Reply:
Suraj "tuned" the long blog post on v1-4, so I think let's leave it there for now